src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -1,32 +1,44 @@
 # Import packages
+import shutil
 import reaction_data as rxd
 import tendl_processing as tp
 import groupr_tools
+from pandas import DataFrame
 
 def main():
     """
     Main method when run as a command line script.
     """
 
+    TAPE20 = 'tape20'
+    TAPE21 = 'tape21'
+
     mt_dict = rxd.process_mt_data(rxd.load_mt_table('mt_table.csv'))
 
-    endf_path = 'tape20'
-    pendf_path = 'tape21'
+    cumulative_data = []
+    for isotope, file_properties in tp.search_for_files().items():
+        element = file_properties['Element']
+        A = file_properties['Mass Number']
+        endf_path, pendf_path = file_properties['File Paths']
+        shutil.copy(endf_path, TAPE20)
+        shutil.copy(pendf_path, TAPE21)
```

Comment:
I think this should work from Pathlib (trying to reduce number of dependencies, even if they're built in; I also think it's a more modern solution???):
```suggestion
        Path(TAPE20).write_bytes(endf_path.read_bytes())
        Path(TAPE21).write_bytes(pendf_path.read_bytes())
```